source/installation/install-docker.txt

```diff
@@ -0,0 +1,136 @@
+.. _install-docker:
+
+=================================================
+Install with a Containerized Cluster Using Docker
+=================================================
+
+.. default-domain:: mongodb
+
+.. facet::
+   :name: genre
+   :values: tutorial
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+You can install Relational Migrator with Kafka in a containerized environment 
+using `Docker <https://docs.docker.com/engine/>`__. This method configures Kafka 
+to store data locally in Docker containers. This deployment method is ideal for 
+educational purposes. Use this if you are looking to learn and understand how to 
+configure your own multi-server Kafka environment. 
+
+.. note::
+
+   This deployment method is not recommended for production workloads. 
+
+About this Task
+---------------
+
+- Relational Migrator is available port 8080.
+
+- In the Docker container, Relational Migrator is installed to 
+  ``/opt/mongodb-relational-migrator/``.
+
+- When running in Docker, the ``user.properties`` file is ignored. Define any 
+  property overrides using environment variables. For example: 
+
+  .. code-block::
+
+     docker run -p 8080:8080 -e MIGRATOR_APP_TELEMETRY_ENABLE=false public.ecr.aws/v4d7k6c9/relational-migrator:latest
+
+- `Docker volumes <https://docs.docker.com/storage/volumes/>`__ can be used for 
```

Comment:
Suggest taking these out of list format and into normal paragraphs

Reply:
Thank you but just for now I might keep them in a list due to the fact that they are all kind of varying on subjects but pending any removal of bullet points I will go ahead and place them into paragraphs.

Reply:
That makes sense if you're keeping the other info!